server/src/constants/format.js

```diff
@@ -0,0 +1,3 @@
+export const DATE_FORMAT = 'DD-MM-YYYY';
```

Comment:
why use this format the database uses the "YYYY-MM-DD" format

Reply:
Solved in a meeting.